run pylint script

adrianherrera opened the pull request.

Commit:
pylint: added pylint script

This script runs pylint with some additional string checkers.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/S2E/build-scripts?pullRequest=7) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/S2E/build-scripts?pullRequest=7) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/S2E/build-scripts?pullRequest=7) it.</sub>

adrianherrera commented:
Made some changes - instead of just providing a pylintrc file I provided a script that a user can run. This script also contains my custom string checkers.

vitalych commented on pylint_rc:
```diff
@@ -0,0 +1,280 @@
+[FORMAT]
+
+# Maximum number of characters on a single line.
+max-line-length=80
```
120 :)

vitalych commented on run_pylint.py:
```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
```
Add copyright notice

adrianherrera commented:
I added the copyright text. I took some code from pylint itself, which is GPL. So I've added a GPL header. If this is correct then I'll merge.

I still think the 80 char limit is the standard for Python code (PEP8, Google style guide, etc.), but I'm not fighting this battle anymore. Here's my white flag 🏳️
